config/jobs/kubernetes/sig-storage/sig-storage-kind.yaml

```diff
@@ -97,6 +97,54 @@ presubmits:
             cpu: "2"
             memory: "6Gi"
 
+  # This jobs runs e2e.test with a focus on tests for vac on a kind cluster
+  - name: pull-kubernetes-e2e-storage-kind-vac
```

Comment:
Could we extend https://github.com/kubernetes/test-infra/blob/39a4617569a8a891eb3d8d633652ed9928aa1f7f/config/jobs/kubernetes/sig-storage/sig-storage-kind.yaml#L149 instead?

Reply:
I don't think you'd even have to extend it, the same feature gate is enabled in that job and the filter is `FeatureGate:VolumeAttributesClass && Slow` so it won't skip the new feature label, it doesn't attempt to skip any `Feature:`

Reply:
Considering this is a GA feature, it should be always triggered. This is a temporary job, we should delete it once the csi driver is updated.

Reply:
updated.